spec/cypress/integration/admin/administrators/create-administrator.js

```diff
@@ -10,15 +10,15 @@ describe("Admin user creating another admin user", () => {
     const email = "j.smith@example.com";
 
     cy.visit(basePath);
-    cy.get("[data-test=create-admin-button").click();
+    cy.get("[data-test=create-admin-button]").click();
```

Comment:
Huh, this was passing, right? Sometimes Cypress surprises me.

Reply:
browsers are very lenient

Reply:
That's the browser, not Cypress. That's behaviour of `document.querySelector`

Reply:
Makes sense - that's a little less surprising considering what other things they let you get away with (broken html? no worries, lemme fix that for you :D)